src/i18n/messages/de.messages.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
The file should be named .fr.messages.json, there was a typo in the Feature Request, we have corrected it ;)

Reply:
Ohh..I have corrected that, would be pushing the changes ASAP.